src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -247,10 +247,10 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
 
       setSafeApp(app)
     }
-    if (staticAppsList.length) {
+    if (!isLoading) {
```

Comment:
the only reason we have useAppList hook in this component is `isLoading` flag. Can we get rid of it at all?